internal/cpugroup/cpugroup.go

```diff
@@ -48,7 +48,7 @@ func Create(ctx context.Context, id string, logicalProcessors []uint32) error {
 		LogicalProcessorCount: uint32(len(logicalProcessors)),
 	}
 	if err := modifyCPUGroupRequest(ctx, operation, details); err != nil {
-		return fmt.Errorf("failed to make cpugroups CreateGroup request for details %+v with: %s", details, err)
+		return errors.Wrapf(err, "failed to make cpugroups CreateGroup request for details %+v with", details)
```

Comment:
"with"... what?

Reply:
hahah good catch. This is awkward now as before we were explicitly mimicking what `errors.Wrap`  would output with the ": %s", but the actual error string will be the exact same afaik, but I'm just going to take out the with.

The error string would look like "failed to make cpugroups CreateGroup request for details (insert details here) with: (insert hcs error here)" for both.